packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -56,71 +87,166 @@ export const pageQuery = graphql`
 
 ### About Cards
 
-Cards are a specifically-styled container that group relatable content and actions.
-Card is an extremely flexible container that does not require specific components inside of it,
-allowing an implementer to compose as needed, but elements such as Title, Body,
-and Button or Anchor are commonly used. See Examples for common Card patterns.
-A Card does not currently handle any interactive events such as hover, click or focus,
-though children composed inside of it may commonly have event handlers.
+Cards are specifically-styled containers that group relatable content and actions. Card is an extremely 
+flexible container that does not require specific components inside of it, allowing an implementer to 
+compose as needed, but elements such as Heading, Text, and Button or Anchor are commonly used.
+See [When to use a Card](#when-to-use-a-card) and [Examples](#examples) for common Card patterns and dos/dont’s. A Card does not 
+currently handle any interactive events such as hover, click, or focus, though children 
+composed inside of it may commonly have event handlers.
 
-Acknowledging the flexibility of Card component, there are several non-negotiable
-properties of a Card that differentiate it from a more basic page-layout element, such as Box:
+Acknowledging the flexibility of Card component, there are several non-negotiable properties 
+of a Card that differentiate it from a more basic page-layout element, such as [Box](/utilities/box):
 
 * A background color of $color-background-body.
 * A border width of $border-width-20.
 * A border radius of $border-radius-20.
-* A border color of $color-border.
+* A border color of $color-border-light.
```

Comment:
This breakdown can be removed since they're laid out in the Anatomy section.

Reply:
With that being said, can we link "non-negotiable properties" above to the Anatomy section?

Reply:
Yep, and when the items are removed, this section could be reworded as a paragraph without the bullet points.